t/valgrind/valgrind.supp

```diff
@@ -146,3 +146,35 @@
    obj:/usr/lib64/libgcrypt.so.20.2.5
    ...
 }
+{
+   <issue_5705>
+   Memcheck:Leak
+   match-leak-kinds: possible
+   fun:calloc
+   fun:UnknownInlinedFun
+   fun:allocate_dtv
+   fun:_dl_allocate_tls
+   fun:pthread_create@@GLIBC_2.2.5
+   obj:/usr/lib64/libcuda.so.550.54.15
+   obj:/usr/lib64/libcuda.so.550.54.15
+   obj:/usr/lib64/libcuda.so.550.54.15
+   obj:/usr/lib64/libcuda.so.550.54.15
+   obj:/usr/lib64/hwloc/libcudart.so.12
```

Comment:
this is very specific to the version of CUDA installed. we should try something super generic to catch different versions and future similar errors by replacing all the specific objects with `...`, something like:
```
{
   <issue_5705>
   Memcheck:Leak
   match-leak-kinds: possible
   fun:calloc
   ...
   fun:hwloc_cuda_discover
   ...
   fun:hwloc_topology_load
   ...
}
```
If that works.